src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts

```diff
@@ -0,0 +1,384 @@
+      expect(componentInstance.progressItems.toArray()[1].isComplete).toBeFalsy();
+
+      nextButton.click();
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+
+      nextButton.click();
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+
+      resetButton.click();
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+
+      expect(componentInstance.activeIndex).toBe(0);
+      expect(componentInstance.progressItems.first.isActive).toBeTruthy();
+      expect(componentInstance.progressItems.first.isComplete).toBeFalsy();
+
+      expect(componentInstance.progressItems.toArray()[1].isActive).toBeFalsy();
+      expect(componentInstance.progressItems.toArray()[1].isComplete).toBeFalsy();
+
+      expect(fixture.componentInstance.resetWasClicked).toBeTruthy();
+    }));
+
+    it('should use inputted button type', fakeAsync(() => {
+      fixture.componentInstance.previousButtonType = 'next';
+      fixture.detectChanges();
+      tick();
+
+      expect(fixture.componentInstance.navButtons.first.buttonType).toBe('next');
+    }));
+
+    it('should default button type to next', fakeAsync(() => {
+      fixture.componentInstance.previousButtonType = undefined;
+      fixture.detectChanges();
+      tick();
+
+      expect(fixture.componentInstance.navButtons.first.buttonType).toBe('next');
+    }));
+
+    it('should use inputted button text', fakeAsync(() => {
+      fixture.componentInstance.previousButtonText = 'good text';
+      fixture.detectChanges();
+      tick();
+
+      expect(fixture.componentInstance.navButtons.first.buttonText).toBe('good text');
+    }));
+
+    it('should default button text for each button type', fakeAsync(() => {
+      fixture.detectChanges();
+      tick();
+
+      expect(fixture.nativeElement.querySelector('#previous-btn button').innerText.trim()).toBe('Previous');
+      expect(fixture.nativeElement.querySelector('#next-btn button').innerText.trim()).toBe('Next');
+    }));
+
+    it('should use inputted disabled state', fakeAsync(() => {
+      fixture.componentInstance.previousButtonDisabled = true;
+      fixture.componentInstance.nextButtonDisabled = true;
+      fixture.componentInstance.resetButtonDisabled = true;
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+      let buttonElements = fixture.nativeElement.querySelectorAll('sky-progress-indicator-nav-button button');
+      let resetButton = fixture.nativeElement.querySelector('#reset-btn button');
+
+      expect(fixture.componentInstance.navButtons.first.disabled).toBeTruthy();
+      expect(buttonElements[0].disabled).toBeTruthy();
+
+      expect(fixture.componentInstance.navButtons.last.disabled).toBeTruthy();
+      expect(buttonElements[1].disabled).toBeTruthy();
+
+      expect(fixture.componentInstance.resetButton.disabled).toBeTruthy();
+      expect(resetButton.disabled).toBeTruthy();
+    }));
+  });
+});
```

Comment:
I'm not seeing any accessibility tests in here. Did they get moved over?

Reply:
missed it, added now though